clients/lsp-eslint.el

```diff
@@ -267,7 +267,7 @@ source.fixAll code action."
   :activation-fn (lambda (filename &optional _)
                    (when lsp-eslint-enable
                      (or (string-match-p (rx (one-or-more anything) "."
-                                             (or "ts" "js" "jsx" "tsx" "html" "vue"))
+                                             (or "ts" "js" "jsx" "tsx" "html" "vue")eol)
```

Comment:
can you use `eos` instead of `eol`?

Reply:
Indeed, it's my first encounter with `rx`, didn't know `eos` exists.